spec/components/finance/statements/npq_statement_selector_spec.rb

```diff
@@ -7,8 +7,9 @@
   let(:npq_lead_provider)    { cpd_lead_provider.npq_lead_provider }
   let!(:npq_statement)       { create(:npq_statement, cpd_lead_provider:) }
   let!(:other_npq_statement) { create(:npq_statement) }
+  let(:cohorts) { Cohort.where(start_year: 2021..) }
```

Comment:
Maybe we should add this as a scope on `Cohort` as seems like we're adhering to this rule in many places, what do you think?

Reply:
if we did not sure what we would name it though?

Reply:
naming is hard 😅 , `active_start_years`? depends what we see 2020 as and what term we use to name that cohort

Reply:
i guess it would be something like `inauguration_onwards` or `inception_onwards`?

Reply:
Maybe, let's do this later when we revisit cohorts further and agree a name with the team